lib/launch.c

```diff
@@ -274,6 +274,8 @@ launch_gatekeeper(void)
 	if (ret != 0)
 		return -1;
 
+	log_ratelimit_enable();
```

Comment:
The library `lib/launch.c` is very simple because all Gatekeeper's stuff is left out of it. We should keep it this way.

Call `log_ratelimit_enable()` at the end of `finalize_stage2()` instead.